corehq/apps/user_importer/importer.py

```diff
@@ -771,10 +858,13 @@ def remove_invited_web_user(domain, username):
     invitation.delete()
 
 
-def remove_web_user_from_domain(domain, user, username, upload_user, is_web_upload=False):
+def remove_web_user_from_domain(domain, user, username, upload_user, user_change_logger=None,
```

Comment:
when would `user_change_logger` be None?

Reply:
`remove_web_user_from_domain` can be None when there is no corresponding web user for a mobile worker.
https://github.com/dimagi/commcare-hq/blob/1779b9e431b184206472311a326d2b02732c76fd/corehq/apps/user_importer/importer.py#L602

Similarly for other places where logger can be None.

The importer logic shuffles between Invitation and users for taking necessary actions. And the logging is only for users and not for invitations, hence the unfortunate branching with logger being `None`.